WebRoot/css/zm.css

```diff
@@ -3757,7 +3792,11 @@ HTML>BODY .appt-selected .appt_allday_body
 
 /* Table for a section of prefs (head + body) */
 .ZOptionsSectionTable {
-	margin-bottom:1.5em;
+	@PrefSectionContentTable@
+}
+
+.ZOptionsSectionTable * {
```

Comment:
do we need a * selector here, i mean if we assign font size to .ZOptionsSectionTable then wouldn't all children of that element inherit the font size?